tldw_Server_API/app/core/Persona/archetype_loader.py

```
@@ -0,0 +1,89 @@
+        try:
+            raw = yaml_file.read_text(encoding="utf-8")
+            data = yaml.safe_load(raw)
+            if not isinstance(data, dict) or "archetype" not in data:
+                logger.warning(
+                    "Skipping {}: missing top-level 'archetype' key", yaml_file.name
+                )
+                continue
+            template = ArchetypeTemplate(**data["archetype"])
+            _CACHE[template.key] = template
+            logger.debug("Loaded archetype '{}' from {}", template.key, yaml_file.name)
+        except Exception:
```

Ruff (0.15.9)

[warning] Do not catch blind exception: `Exception`

(BLE001)